app/bundles/EmailBundle/Form/Type/ScheduleSendType.php

```diff
@@ -0,0 +1,127 @@
+ * @extends AbstractType<ScheduleSendType>
+ */
+final class ScheduleSendType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add(
+            'publishUp',
+            DateTimeType::class,
+            [
+                'widget' => 'single_text',
+                'label'  => 'mautic.email.send.datetime.start',
+                'attr'   => [
+                    'class'       => 'form-control',
+                    'data-toggle' => 'datetime',
+                ],
+                'format'      => 'yyyy-MM-dd HH:mm',
+                'required'    => false,
+                'html5'       => false,
+                'constraints' => new NotBlank(
+                    [
+                        'message' => 'mautic.core.value.required',
+                    ]
+                ),
+            ]
+        );
+
+        $builder->add(
+            'continueSending',
+            YesNoButtonGroupType::class,
+            [
+                'label'    => 'mautic.email.send.continue',
+                'required' => false,
+                'attr'     => [
+                    'tooltip' => 'mautic.email.send.continue.tooltip',
+                ],
+                'data'     => $options['data']['continueSending'] ?? false,
+            ]
+        );
+
+        $builder->add(
+            'publishDown',
+            DateTimeType::class,
+            [
+                'widget' => 'single_text',
+                'label'  => 'mautic.email.send.datetime.end',
+                'attr'   => [
+                    'class'            => 'form-control',
+                    'data-toggle'      => 'datetime',
+                    'data-show-on'     => '{"schedule_send_continueSending_1":"checked"}',
+                ],
+                'format'      => 'yyyy-MM-dd HH:mm',
+                'html5'       => false,
+                'required'    => false,
+            ]
+        );
+
+        if (empty(array_filter($options['data'] ?? []))) {
+            $builder->add(
+                'buttons',
+                FormButtonsType::class,
+                [
+                    'save_text'  => 'mautic.email.send.schedule',
+                    'save_icon'  => 'ri-calendar-schedule-fill',
+                    'apply_text' => false,
+                ]
+            );
+        } else {
+            $builder->add(
+                'buttons',
+                FormButtonsType::class,
+                [
+                    'save_text'   => 'mautic.email.send.schedule.update',
+                    'save_icon'   => 'ri-calendar-schedule-line',
+                    'save_class'  => 'btn btn-primary',
+                    'apply_text'  => 'mautic.email.send.schedule.cancel',
+                    'apply_icon'  => 'ri-close-line',
+                    'apply_class' => 'btn btn-secondary btn-cancel',
+                    'cancel_text' => 'mautic.core.close',
```

Comment:
```suggestion
                    'cancel_text' => false,
```

I suggest removing the close button in favor of the X already present on the modal screen

<img width="116" height="91" alt="image" src="https://github.com/user-attachments/assets/419b1ed9-9f46-4b31-a408-738a89562252" />

Modal would look like:
<img width="627" height="352" alt="image" src="https://github.com/user-attachments/assets/13a32cf7-73d6-40df-b80a-89859d8108e6" />

Reply:
Cancel schedule does not close the modal only.
Cancel schedule removes any previously set schedule on one click.

Reply:
Yeah, keep "Cancel schedule" and "Update schedule" but remove "Close"

Reply:
And how do you expect the modal to exist without any changes? I don't understand.

Reply:
I understand the comment provided, the `X` cross should be enough, but this Cancel button is everywhere on modal.
Let's keep constitency.
<img width="1249" height="635" alt="image" src="https://github.com/user-attachments/assets/42456705-c0e3-421b-aba4-a30e7e9ac698" />